src/hdinsightonaks/azext_hdinsightonaks/tests/latest/test_clusterpool_operations.py

```diff
@@ -54,4 +55,4 @@ def test_clusterpool_operations(self):
         ])
 
         # Delete a Cluster Pool.
-        self.cmd('az hdinsight-on-aks clusterpool delete -g {rg} -n {poolName} --yes')
+        # self.cmd('az hdinsight-on-aks clusterpool delete -g {rg} -n {poolName} --yes')
```

Comment:
May I ask why this command is commented out? If it is cannot be tested for now, it is better to add some explanatory comments or remove them.

Reply:
Added comments.